internal/lookoutingester/instructions/instructions_test.go

```diff
@@ -347,14 +347,10 @@ func TestReprioritised(t *testing.T) {
 
 func TestInvalidEvent(t *testing.T) {
 
-	// This event is invalid as it references a job that doesn't exist
-	nonExistingJob, _ := armadaevents.ProtoUuidFromUlidString("01f3j0g1md6qx7z5qb148qnh4r")
```

Comment:
We stopped processing `JobRunLeased` messages a while back which broke the purpose of this test.  It should be ignoring the event because its invalid , instead it was now ignoring the event because we don't event parse it.

I've changed it to use a `JobRunRunning` event, which we do care about.